### PR TITLE
Clip hru shapefile to coterminous US

### PR DESCRIPTION
**Pull request by mhines-usgs**
for https://github.com/usgs-makerspace/makerspace-sandbox/issues/220

### CODE REVIEW

**Comment by mhines-usgs** [comment]
tested this locally and it works 🎉 hopefully the shp i uploaded to s3 and the modifications to the jenkinsfile are correct, haven't tested those yet but will on test.

File: jenkins/hru_R_Jenkinsfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ pipeline {
                           userRemoteConfigs: [[url: 'https://github.com/usgs-makerspace/wbeep-processing']]
                         ])
         sh 'aws s3 cp --recursive s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/GF_nat_reg.gdb cache/GF_nat_reg.gdb'
+		sh 'aws s3 cp --recursive s3://prod-owi-resources/resources/Application/wbeep/${TIER}/coterminous_us . --include "*"'
```
Comment (wdwatkins) [inline jenkins/hru_R_Jenkinsfile]: TIL that coterminous and conterminous are both (equivalent) words 🤔

File: src/process_hru_shapes.R [hunk 2]
```diff
@@ -4,7 +4,7 @@ library(lwgeom)
 proj_string <- 4326
 
 gfdb <- "cache/GF_nat_reg.gdb"
-hru_reduced <- read_sf(gfdb, "nhru")  %>%
```
Comment (wdwatkins) [inline src/process_hru_shapes.R]: Just curious, is this an equiavlet field?

Comment (mhines-usgs) [reply src/process_hru_shapes.R]: oh woops, for some reason the gdb i have has a different name, but no, the one on the server is nhru, will change back!

File: jenkins/hru_R_Jenkinsfile (hunk 1 above)
Comment (mhines-usgs) [reply jenkins/hru_R_Jenkinsfile]: huh! i never heard that other term before! lol

Comment (mhines-usgs) [reply jenkins/hru_R_Jenkinsfile]: this version is currently in higher fashion ;) 
image

Comment (mhines-usgs) [reply jenkins/hru_R_Jenkinsfile]: image

Comment (wdwatkins) [reply jenkins/hru_R_Jenkinsfile]: Haha.  Guess I'm behind the times 👴

File: src/process_hru_shapes.R (hunk 2 above)
Comment (wdwatkins) [reply src/process_hru_shapes.R]: Maybe that is Roland's newer fabric or something that you have locally?

Comment (mhines-usgs) [reply src/process_hru_shapes.R]: ah, that could very well be actually! makes sense based on the date I downloaded it

**Pull request merged**